scripts/npm/link/index.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+var path = require( 'path' );
+var fs = require( 'fs' );
+var componentsDir = path.join( __dirname, '..', '..', '..', 'components' );
```

Comment:
I'm slightly worried about this breaking if we change the directory structure of the site, but it's not a huge deal.